docs/openapi/components/schemas/common/AgParcelDelivery.yml

```diff
@@ -216,7 +216,71 @@ example: |-
       "phoneNumber": "555-895-1661",
       "faxNumber": "555-497-2527"
     },
-    "AgPackage": [],
+    "AgPackage": [
+      {
+        "type": [
+          "AgPackage"
+        ],
+        "packageName": "Avocados, Bulk",
+        "grade": "AA",
+        "responsibleParty": {
+          "type": "Entity",
+          "name": "Example Responsible Party Organization",
+          "email": "Chadrick_Gibson@example.com",
+          "phoneNumber": "+1-555-820-1520",
+          "entityType": "Organization"
```

Comment:
This is doing to result in a lot of undefined terms... which will make your RDF not crawl-able.

Reply:
I don't understand, could you elaborate?

The only change I made was `$ref: ./Product.yml` to `$ref: ./AgPackage.yml`, which given the context I assumed was just fixing a mistake.

Reply:
pretty much you should not be increasing the number of undefined terms in your PR.

Reply:
I'm seeing similar usage of examples with type, name, email, phoneNumber, entityType throughout the vocab. For instance, in BillOfLading (line 176):
```
    "carrier": {
      "type": "Entity",
      "name": "Carrior Goods And More",
      "email": "Adaline29@example.com",
      "phoneNumber": "+1 555-234-9983",
      "entityType": "Organization"
    },
```

I think I understand the issue you're raising but that seems out of scope for this PR. For instance, the block of code above is included as part of an AgPackage example property that was lifted directly from AgPackage, and I don't want to change AgPackage in this PR. I think it would make more sense to create a separate issue to update these entity examples across the board.

So would it work to leave this as-is and create a separate issue?

Reply:
For some reason entityType is not being picked up on the Entity objects, I'm not sure why that is.

Reply:
That seems like its own issue though, not a blocker on this PR

Reply:
Ignore my comments i see you have #440 to fix this